taui/src/index.js

```diff
@@ -163,6 +163,18 @@ class InitializationWrapper extends React.Component {
     }
   }
 
+  // TODO: remove this later
+  // This is to test and prove that the local Django API works
+  componentDidMount () {
+    fetch('http://localhost:9966/api/').then(function (response) {
```

Comment:
No need to change this if you're going to remove this whole thing, but this can be just `/api/` since this app isn't operating cross-origin any longer:
```suggestion
    fetch('/api/').then(function (response) {
```